src/features.ts

```diff
@@ -24,3 +30,15 @@ export const DefaultFeatureSetForTesting: Features = {
 
 export const FeatureEnabled = (feature: Feature): boolean =>
   feature === Feature.Enabled;
+
+const boolToEnabled = (currentState: boolean): Feature => (currentState ? Feature.Enabled : Feature.Disabled);
+
+// @TODO: build features from settings
+export const FeaturesFromSettings = (vortexApi: VortexApi): Features => {
+  const state: types.IState = vortexApi.store.getState();
+  return {
+    REDmodding: boolToEnabled(util.getSafe(state, [`settings`, `redmod`, `redmodEnable`], false)),
```

Comment:
Minor: I'd still be tempted to further namespace these to us. I dunno if it helps, but I don't trust the store.